README.md

```diff
@@ -29,7 +29,7 @@ If you are having some issues please checkout [wiki](https://github.com/facebook
 
 ## Note
 
-[appium-webdriveragent](https://www.npmjs.com/package/appium-webdriveragent) on npm registry expects build via [XCUITest driver](https://github.com/appium/appium-xcuitest-driver).
+[appium-webdriveragent](https://www.npmjs.com/package/appium-webdriveragent) which is available via `npm` does not have `Script` directory. [XCUITest driver](https://github.com/appium/appium-xcuitest-driver) does the same thing instead. Please git clone this repository and follow the getting started if you would like to build WDA environment by manual.
```

Comment:
... and follow `Getting Started` tutorial ... to build WDA environment manually

Reply:
The name of the folder is Scripts

Reply:
will remove here since the script will back
https://github.com/appium/WebDriverAgent/pull/192#pullrequestreview-268412413